src/internal/m365/collection/site/backup.go

```diff
@@ -171,6 +172,8 @@ func CollectLists(
 			continue
 		}
 
+		lmt := list.GetLastModifiedDateTime()
```

Comment:
nit: `modTime` instead of `lmt` might be more descriptive.

Reply:
made the change in #4945